training/src/anemoi/training/schemas/diagnostics.py

```diff
@@ -283,6 +287,8 @@ class MlflowSchema(BaseModel):
     "Specifies the maximum number of retries for MLflow HTTP requests, default 35."
     max_params_length: int = MAX_PARAMS_LENGTH
     "Maximum number of hpParams to be logged with mlflow"
+    save_dir: str | None = None
```

Comment:
For this we'd need an entry to the config linked to hardware.paths.logs.mlflow as at the moment this is hardcoded and not configurable